en/concepts/scene/shadow.md

```diff
@@ -46,22 +46,46 @@ Adjust the direction of the directional light to adjust the position of the shad
 
 ## ShadowMap
 
-ShadowMap renders the scene with the light source as the viewpoint. From the position of the light source, the places in the scene that are not visible are where the shadows are created.
+| Properties | Explanation |
+| --- | --- |
+| **Enabled**     | Whether to turn on the shadow effect |
+| **Type**        | Choose shade type |
+| **ShadowColor** | Color value of the resulting shadow |
+| **Normal**      | Normals to vertical and shadow planes |
+| **Distance**    | The distance of the shadow plane from the coordinate origin in the direction of the normal |
 
 ![Shadow Map Panel Details](shadow/shadowmap-properties.png)
 
-| Property | Description |
-| :--- | :--- |
-| **Enabled** | Whether to enable the shadow effect |
-| **Type** | Shadow type | 
-| **ShadowColor** | Shadow color |
-| **Pcf** | Set the anti-aliasing level of the shadow edge, currently including **HARD**, **FILTER_X5**, **FILTER_X9**, **FILTER_X25** |
-| **Near** | Set the near clip plane of the main light source camera |
-| **Far** | Set the far clip plane of the main light source camera |
-| **OrthoSize** | Set the orthogonal viewport size of the main light source camera |
-| **ShadowMapSize** | Set the texture size of the shadow |
-| **Aspect** | Set the aspect ratio of the orthogonal viewport of the main light source camera |
+### PCF soft shadow support
+
+* Percentage asymptotic filtering (PCF) is a simple and common technique to reverse shadow edges. It performs sampling around the fragment, then calculates the ratio of the sample closer to the light source than the fragment, uses this ratio to scale the scattered light and specular light components, and then colorizes the fragment. After using this technique, the shadow edges appear to be blurred.
+* At present, 5x, 9x, and 25x sampling are used to attenuate the coloring. As the sampled area expands, the shadow edge will have a blur effect.
+
+### AutoAdapt adaptive shadow calculation technology
+
+Automatically calculate the range of shadows under the viewport and the distance of the shadow camera.
+
+### GPU Instancing support
+
+We can remove all these static objects such as vegetation from the scene, and save their position, zoom, uv offset and other related information. When rendering is needed, use the saved information to render through Instance, which can reduce The rendering time of large batches of the same objects that cannot be batched due to memory reasons. Generate a shadowMap for the statically batched model, and sample it where it is used.
```

Comment:
```suggestion
When there are large batches of same objects in the project, but cannot be batched due to memory reasons, it will take too long to render. In this case, you can save information about the position, scaling and uv offset of these static objects (e.g. vegetation, etc.) and then exclude them all from the scene, and render them by `Instance` when you need to render them based on the saved information.
```